assets/src/domain/blocks/event-attendees/controls/SelectOrder.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { SelectControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+import { AttendeesEditProps } from '../types';
+
+interface SelectOrderProps {
+	setOrder?: (order: string) => void;
+	order: string;
+}
+
+const SelectOrder: React.FC<AttendeesEditProps> = ({ attributes, setAttributes }) => {
```

Comment:
a generic control for selecting the order for a list will also be highly reused, 
so let's rename/relocate this to something like:

```
assets/src/domain/blocks/components/SortOrderControl
```

or

```
assets/src/domain/blocks/components/SortOrderSelect
```

or

```
assets/src/domain/blocks/components/SortOrderSelector
```